selfapi.js

```diff
@@ -367,6 +374,24 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  buildIndexRoutes: function (basePath) {
+      var fullPath = normalizePath(this.path, basePath) || '/';
+      this.get({
+          title: 'Index',
+          handler: (request, response) => {
+              var routes = {};
+              Object.keys(this.children).forEach(child => {
+                  if (child == "null") {
+                      return;
+                  }
+                  routes[child.replace(/^\//, '')] = nodepath.join(fullPath, child);
```

Comment:
Nit: Why remove the initial `/`? Is the convention for other API-index-like features to remove it? In general, I prefer to keep it, except if there is a good reason not to.